src/main/java/mate/academy/Main.java

```diff
@@ -21,6 +22,8 @@ public static void main(String[] args) {
 
         // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
```

Comment:
It's good practice to handle exceptions that may occur during the execution of the CompletableFuture. You might want to add exceptionally or handle methods to handle potential exceptions.